src/containers/sort/utils.js

```diff
@@ -23,14 +33,18 @@ export const fetchSortedData = ({ data = [], columnType, columnName, direction }
 
     case 'string':
     case 'singleselect':
-      let firstValue, secondValue;
       return data.sort((a, b) => {
-        firstValue = a[columnName] || '';
-        secondValue = b[columnName] || '';
-        if (typeof firstValue === 'number' || typeof secondValue === 'number') {
-          return isAscending ? +firstValue - +secondValue : +secondValue - +firstValue;
+        const foundValue = a[columnName];
```

Comment:
Please crosscheck this sorting bit. I think changing the previous one to work with object is enough .